containers/ecr-viewer/src/app/components/DetailsSidePanel.tsx

```diff
@@ -128,7 +128,7 @@ export const DetailsSidePanel = ({
           <div className="border-bottom border-base-lightest">
             <ModalHeading
               id={`details-sidepanel-${id}-heading`}
-              className="font-sans-3xl margin-bottom-0"
+              className="font-sans-3xl margin-bottom-0 overflow-wrap-break-word"
```

Comment:
out of curiosity, why break word here and not anywhere?

also out of curiosity, would it be insanity to just add a rule like this to our css instead of being targetted?

```
* {
  overflow-wrap: anywhere;
}
```